crates/eww/src/app.rs

```diff
@@ -358,6 +359,9 @@ fn initialize_window(
         }
         display_backend::set_xprops(&window, monitor_geometry, &window_def)?;
     }
+
+    window.show_all();
+
```

Comment:
So... it's actually pretty simple in hindsight :sweat_smile: 

Thanks again @Airblader for helping out, I learned something today :)